src/ring_pedersen.rs

```diff
@@ -0,0 +1,318 @@
+use crate::paillier::DecryptionKey;
```

Comment:
todo: Add module-level docs (at least a one-line description, and optionally more context about commitment schemes and expected usage of this module) and a copyright notice.

Reply:
What copyright notice should I use? Presumably it shouldn't be the facebook one, should it be:
```
// Copyright (c) Bolt Labs, Inc.
//
// This source code is licensed under both the MIT license found in the
// LICENSE-MIT file in the root directory of this source tree and the Apache
// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
// of this source tree.
```

Reply:
Yeah, that looks good. Add 2023 after the `(c)`, please.